Neos.ContentRepository.TestSuite/Classes/Behavior/Features/Bootstrap/ProjectedNodeTrait.php

```diff
@@ -265,7 +266,15 @@ public function iExpectThisNodeToExactlyInheritTheTags(string $tagList): void
 
     protected function initializeCurrentNodeFromContentGraph(callable $query): void
     {
-        $this->currentNode = $query($this->currentContentRepository->getContentGraph());
+        $contentGraphFinder = $this->currentContentRepository->projectionState(ContentGraphFinder::class);
+        $contentGraphFinder->forgetInstances();
+        if (isset($this->currentContentStreamId)) {
+            $contentGraph = $contentGraphFinder->getByWorkspaceNameAndContentStreamId($this->currentWorkspaceName, $this->currentContentStreamId);
+        } else {
+            $contentGraph = $this->currentContentRepository->getContentGraph($this->currentWorkspaceName);
+        }
+        // FIXME: query->workspaceName this _might_ be wrong as the query could use a different workspace/contentstream
+        $this->currentNode = $query($contentGraph);
```

Comment:
i dont fully understand the todo

Reply:
removing, doesn't matter anymore, I think it was more a marker for me during refactoring.